src/pages/sighting/SightingEntityHeader.jsx

```diff
@@ -130,6 +137,12 @@ export default function SightingEntityHeader({
                   disabled: pending || sightingIsReviewed,
                   label: 'Mark sighting reviewed',
                 },
+                {
+                  id: 'rerun-identification',
+                  onClick: () => setRerunDialogOpen(true),
+                  disabled: pending,
```

Comment:
would like to add some more nuance here, like disable the button if identification is currently in progress.